SonarQube.Common/ProcessRunner.cs

```diff
@@ -60,6 +63,12 @@ public bool Execute(string exeName, string args, string workingDirectory, int ti
             {
                 throw new ArgumentNullException("logger");
             }
+            if (!File.Exists(exeName))
+            {
+                logger.LogError(Resources.ERROR_ProcessRunner_ExeNotFound, exeName);
+                this.ExitCode = ErrorCode;
+                return false;
+            }
```

Comment:
Fit and finish: improved the user experience if we try to start an exe that can't be found. This can happen e.g. if you run the bootstrapper from the command line in post-processor mode without first having run it in pre-processor mode. Previously you would get an unhelpful exception; now you get a short error message.
I've added a unit test.